spec/unit/local_system_spec.rb

```diff
@@ -102,33 +102,23 @@
       }.not_to raise_error
     end
 
-    it "raises Machinery::Errors::IncompatibleHost on hosts that can not run machinery" do
+    it "puts an IncompatibleHost message on hosts that can not run machinery" do
       allow(LocalSystem).to receive(:os).and_return(OsSles11.new)
+      LocalSystem.validate_machinery_compatibility
 
-      expect {
-        LocalSystem.validate_machinery_compatibility
-      }.to raise_error(Machinery::Errors::IncompatibleHost)
+      expect(captured_machinery_output).to include(
+        "Running Machinery is not supported on this system.\nSupported operating systems are:"
+        )
```

Comment:
I would add an additional expect for at least one supported OS which won't change in the nearby future like SLES12